src/v2/Apps/Order/Routes/Status/index.tsx

```diff
@@ -233,9 +234,41 @@ export class StatusRoute extends Component<StatusProps> {
     )
   }
 
+  shouldButtonDisplay(): React.ReactNode | null {
+    const {
+      match,
+      order: { stateReason },
+    } = this.props
+    const isModal = !!match?.location.query.isModal
+    const declinedStatuses = [
+      "buyer_rejected",
+      "seller_rejected_offer_too_low",
+      "seller_rejected_shipping_unavailable",
+      "seller_rejected",
+      "seller_rejected_artwork_unavailable",
+      "seller_rejected_other",
+    ]
+    const isDeclined = declinedStatuses.includes(stateReason!)
+
+    if (isModal || isDeclined) {
+      return null
+    }
+
+    return (
+      <Button
+        onClick={() => {
+          window.location.href = "/"
+        }}
```

Comment:
```tsx
// @ts-ignore
as={RouterLink}
to="/"
```

The types for polymorphic `as` are weird so that's what's necessitating the `@ts-ignore`.

In general you never want to do a link like this since it breaks shift+cmd click, etc. (You can of course handle that if necessary but there's no need here.)

Reply:
(I see this is just moved from below but might as well fix it anyway.)